### ISSUE TITLE
Wording on IQR info box.

### ISSUE THREAD

**Issue by catherinecarter** [phetsims/center-and-variability#167]
The info box currently shows the IQR definition as shown in the image below:
<img width="789" alt="image" src="https://user-images.githubusercontent.com/109630002/235746619-6c779a78-3f5a-450f-87e4-90c14082dae7.png">

Please change the wording of the IQR definition to be as shown in the image below. In the definition of the IQR, I'd prefer it to say, "... the distance between quartile 1 and quartile 3," but I'm not sure it will fit. If not, please make it as below:
![image](https://user-images.githubusercontent.com/109630002/235746860-175b20c7-9f42-4551-b762-6bc412e3f1c4.png)

**Comment by catherinecarter** [comment]
Yep, I agree. My only thought is moving too far down into the area where the plot might be with points, we'll need to be conscious of potential overlap.

**Comment by catherinecarter** [comment]
Here are some design changes to make the info box more clear, and a mockup of a potential solution for when the labels overlap.
First, design changes: 
- [ ] Ovals highlighting the Q1 and Q3 in both the list of numbers and above the box are now slightly rounded rectangles (mine have a pixel radius of 3)
- [ ] The blue rectangle is only highlighting the box rather than spanning all the way to the axis
- [ ] The entire boxplot and labels are closer to the number line to allow for space under the calculations with no overlap or occlusion
- [ ] The Q1 and Q3 labels above the boxplot are above the median in case these values overlap
- [ ] The line under the median is now an arrow above

![image](https://github.com/phetsims/center-and-variability/assets/109630002/c10f8bf9-cb1a-46b9-acab-de82479af1a7)


And an idea for when there are many things overlapping. Is it possible to detect overlap and then stack the labels vertically from lowest value on the bottom to the highest value on the top? I also am only showing 1 arrow. Not sure how that would or could work.
![image](https://github.com/phetsims/center-and-variability/assets/109630002/4fd19cd1-ee86-45d2-9cc3-47ce46b8ef9c)

![image](https://github.com/phetsims/center-and-variability/assets/109630002/71fceda8-b272-403c-967b-928b07810e67)

**Comment by catherinecarter** [comment]
And after thinking it through with Amanda, let's delete the word, "Median," as they will likely know what this colored arrow means by now. Then, we can move the Q1/Q3 labels down a little closer to the box, and if one of Q1/Q3 + Median overlap, there can be a little offset in their overlap (so the distance between the min/Q1/Q3/max arrows can all have the same distance from their respective arrows and the boxplot, while the median arrow touches the boxplot.
![image](https://github.com/phetsims/center-and-variability/assets/109630002/89411098-bebf-4727-ba07-a3fd05d7ad1b)

![image](https://github.com/phetsims/center-and-variability/assets/109630002/23010eeb-9d21-4ed8-9bb5-f1286fa647e7)

I'm thinking if the median overlaps with Q1, the median arrow should go behind Q1, since it's 3rd in the 5 number summary. If the median and Q3 overlap, the median arrow should go in front of the Q3 arrow. This seems logical to indicate the order of the 5-number summary, with the lower numbers being first (on the top layer): min, Q1, median, Q3, max
![image](https://github.com/phetsims/center-and-variability/assets/109630002/01b93ee6-90b1-4aea-b230-38276eee5f30)

![image](https://github.com/phetsims/center-and-variability/assets/109630002/9e27e604-4ad3-43a8-abce-e964c17dcdeb)

I'm open to thoughts about whether my logic seems to be along the right path, as well as how challenging this might be for development. Thanks!

**Comment by matthew-blackman** [comment]
This is ready for a full design review @catherinecarter 🎉

**Comment by catherinecarter** [comment]
Everything looks really great! 
I have one request: when Q1 and Q3 are stacked, is it possible to write the word, "Median," between them? Just on those instances? 
For example, the word, "Median," would appear between the Q3/Q1 labels here: 
![image](https://github.com/phetsims/center-and-variability/assets/109630002/475b4747-63e4-4752-8104-021a55940e0f)

but not here:
![image](https://github.com/phetsims/center-and-variability/assets/109630002/4064bae2-6422-4961-96ed-8470cb6d499b)

Is this possible?

**Comment by catherinecarter** [comment]
Also, I just noticed the blue rectangle no longer goes to the axis in the accordion box. To be consistent with the other measures of spread, I think the rectangle should go to the axis in the accordion box, but since the info box isn't one where you can easily toggle between them per measure of spread, I like that the rectangle only highlights the IQR in the info box.

Is it possible to have the rectangle go to the axis in the accordion box but only highlighting the IQR in the info box?

**Comment by catherinecarter** [comment]
The box to the axis in the accordion box looks good, and the IQR highlight in the info box looks good, too. In terms of the Q1 and Q3 getting close together (assuming you mean this situation below):
![image](https://github.com/phetsims/center-and-variability/assets/109630002/eb872648-a8c1-4977-a49e-a9e16776de32)

I think it's fine. Yes, they're squished, but their positions are still clear. 

The only thing I'm not seeing is the word, "Median," when all the balls are stacked at the same location: 
![image](https://github.com/phetsims/center-and-variability/assets/109630002/8c27ea10-aab8-44ae-8644-7ff20aaec57f)

One thing I noticed just now is the dots sinking into the number line in the info box when "dot plot" is the plot type chosen. Can we lift those so they look like they are sitting on top of the number line?

**Comment by matthew-blackman** [comment]
@catherinecarter the median label is now shown between the Q1 and Q3 labels when they are stacked. Ready for your review!

**Comment by catherinecarter** [comment]
@matthew-blackman - looking at it (even though I had mocked it up), I'm thinking that the median arrow would look better there rather than the word, but if it's too much effort, I think it's great. The reason I'm seeing this is because I neglected to consider there are two median markers now, something I didn't have in my mockups since I made them prior to us removing the word entirely and using the red arrow to replace it.

If you think it's an easy thing to do, can we move the arrow between the Q1 and Q3 in these edge cases so the median marker stayed consistent with the rest of the screens/scenes? This would mean the arrow would change positions _only_ in the case where Q1, median, and Q3 line up on the same number line position. As I said, if it's too much, it looks great, shows where the median is, and removes the strangeness of how Q1 and Q3 were stacked with nothing in between. Something like if, if it's possible:
![image](https://github.com/phetsims/center-and-variability/assets/109630002/c7f9473e-805c-4c9a-b50d-bccbc7b850ec)

**Comment by matthew-blackman** [comment]
This is updated and looks really good! Ready for your review @catherinecarter :)

**Comment by catherinecarter** [comment]
Wow, it looks so good! Thanks for making that change.
